Sources/SwiftFusion/Core/FixedShapeTensor.swift

```diff
@@ -78,6 +78,10 @@ extension FixedShapeTensor {
   }
 
   public static var zero: Self { .init(Tensor(zeros: Self.shape)) }
+
+  public var zeroTangentVectorInitializer: () -> Self {
+    { Self.zero }
```

Comment:
Since uses of `.zero` are a red flag that I'm trying to eliminate, we should factor things differently, e.g. make a common `static var zero_` that `zero` invokes, erwhatever.